python/tests/res/enkf/test_site_config.py

```diff
@@ -65,7 +65,8 @@ def test_constructors(self):
                     [
                         ERT_SHARE_PATH + '/forward-models/res',
                         ERT_SHARE_PATH + '/forward-models/shell',
-                        ERT_SHARE_PATH + '/forward-models/templating'
+                        ERT_SHARE_PATH + '/forward-models/templating',
+                        ERT_SHARE_PATH + '/forward-models/deprecated'
```

Comment:
Just a curiosity. I expected to see ```
ERT_SHARE_PATH + '/forward-models/res',
ERT_SHARE_PATH + '/forward-models/res/deprecated',
ERT_SHARE_PATH + '/forward-models/shell',
ERT_SHARE_PATH + '/forward-models/shell/deprecated',
...
```
Any particular reason why you opted for this?
Overall, I don't think it really matters. In the end we just have a big pool of jobs once they get into libres

Reply:
Because if they are deprecated files they do not deserve their own folder,  and I thought all deprecated files should be ashamed of being deprecated in the same folder.

Joking aside there are several reasons I chose going with one folder:
* reduces complexity when trying to understand what is with the deprecated folder
* makes removing them easier when we completely stop supporting old-style forward-model configs
* it was easier to implement while reducing the chance for errors (i.e putting a file in the wrong folder)

Reply:
I would suggest we do not call the folder `deprecated`. At least not yet. How about `old_style` or similar instead? The thing is that currently there is no realistic alternative, so they are not deprecated yet. They will be, but that is still some time into the future...